app/models/appointment.rb

```diff
@@ -1,7 +1,10 @@
 class Appointment < ApplicationRecord
   include PgSearch
 
-  pg_search_scope :search, against: %i(id first_name last_name)
+  pg_search_scope :search, against: %i(id first_name last_name), associated_against: {
+    guider: [:name]
```

Comment:
They do, but I thought leaving the array form in there would show the next person in this file that they can just add another key if they want, instead of having to read the docs.

Maybe that's dumb though. I'll change it.